src/main/resources/standardTestCases/final_call_logs_dir_local.test

```diff
@@ -0,0 +1,21 @@
+# Tests that a runtime option to save call logs is copying said logs appropriately.
+
+name: final_call_logs_dir_local
+backends: [Local, Tes]
+backendsMode: "any"
+tags: [localdockertest]
```

Comment:
Actually, following @danbills's last update, I think I should be dropping this `backendsMode` completely and having a separate `.test` file for the TES backend.